components/learningUnits/LearningUnitsOfCurriculum.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { useOperation } from "react-openapi-client";
+import { Alert, Container } from "@mui/material";
+import Loader from "../common/Loader";
+import LearningUnit from "./LearningUnit";
+
+export default function LearningUnitsOfCurriculum({ curriculumId }) {
+  const {
+    loading: completedLoading,
+    error: completedError,
+    data: completedData,
+  } = useOperation(
+    "listCompletedLearningUnitsOfCurriculumCompletedByUser",
+    curriculumId
+  );
+
+  const {
+    loading: learningUnitsLoading,
+    data: learningUnitsData,
+    error: learningUnitsError,
+  } = useOperation("listLearningUnitsOfCurriculum", curriculumId);
+
+  // const isCompleted = false;
+
+  if (learningUnitsError || completedError) {
+    return <Alert severity="error">Error</Alert>;
+  }
+  if (learningUnitsLoading || completedLoading) {
+    return <Loader />;
+  }
+
+  const completedLearningUnits = completedData.map(
+    (completedLearningUnit) => completedLearningUnit.learning_unit_id
+  );
+
+  return (
+    <Container maxWidth="sm">
+      {learningUnitsData.map((learningUnit) => (
+        <LearningUnit
+          learningUnit={learningUnit}
```

Comment:
This function is throwing an error message in the browser console:  _Each child in a list should have a unique "key" prop._ 
This could be easily addressed adding `key={learningUnit.id}` in the `LearningUnit` component.

Reply:
Thanks, done!